frameworks/Java/quarkus/hibernate/src/main/java/io/quarkus/benchmark/resource/JsonResource.java

```diff
@@ -1,22 +1,22 @@
 package io.quarkus.benchmark.resource;
 
-import java.util.Map;
-import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.CompletionStage;
-
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
+import java.util.Collections;
+import java.util.Map;
 
 @Path("/json")
 public class JsonResource {
     private static final String MESSAGE = "message";
     private static final String HELLO = "Hello, World!";
+    private static final Map<String, String> map = Collections.singletonMap( MESSAGE, HELLO );
```

Comment:
Instantiating a new object for each request is one of the [requirements](https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#json-serialization).

> i. For each request, an object mapping the key `message` to `Hello, World!` must be instantiated.

Reply:
Oops, missed that. Fixed.